amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-troubleshooting.adoc

```diff
@@ -8,6 +8,14 @@ include::connectors::partial$verbose-exception-logs.adoc[]
 
 include::connectors::partial$verbose-logs-org.adoc[]
 
+== Troubleshoot AWS STS Regionalized Endpoints
+
+Since the upgrade of AWS SDK from v1 to v2, the endpoint that the STS client of the AssumeRoleConnectionProvider uses to get session tokens now is Regionalized instead of Global. Refer to the https://docs.aws.amazon.com/sdkref/latest/guide/feature-sts-regionalized-endpoints.html[AWS STS Regionalized endpoints] documentation for details.
+Additionally, as explained in the https://docs.aws.amazon.com/general/latest/gr/sts.html[AWS Security Token Service endpoints and quotas] documentation, regionalized endpoints shouldn't represent any change in the connector's functionality.
+AWS recommends using Regional STS endpoints to reduce latency, build in redundancy, and increase session token validation.
```

Comment:
AWS recommends using Regional STS endpoints to reduce latency, build in redundancy, and increase session token validation.